common/test/acceptance/tests/lms/test_lms_split_test_courseware_search.py

```diff
@@ -145,15 +116,6 @@ def test_page_existence(self):
         self._auto_auth(self.USERNAME, self.EMAIL, False)
         self.courseware_search_page.visit()
 
-    def test_search_for_experiment_content_user_not_assigned(self):
```

Comment:
The behavior of search is not the same anymore. The user is now assigned content automatically.
See my work on #9292 .